### PR TITLE
Fix tagging pull request versions with latest

### PR DESCRIPTION
**Pull request by gjtempleton**
Currently we tag images with `latest` on a pull request as Travis gives the branch as master, this performs some logical checks as to which tag to use based on branch and event type.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ LABEL org.label-schema.vendor="Skyscanner" \
       org.label-schema.vcs-ref=$GITSHA \
       org.label-schema.build-date=$BUILDDATE
 
+RUN apk update
```
Comment (balintzs) [inline Dockerfile]: What do you need this for?

Comment (gjtempleton) [inline Dockerfile]: I added it for cache busting of the layers above it to test the build and tags were working correctly. Now removed.

**Comment by balintzs** [comment]
This will still create the image with tag `master`. How about we disable publishing on PR all together?

**Comment by gjtempleton** [comment]
@balintzs You alright with this being merged in now? Would like to merge this in then modify #6 to use the same chained steps as this.

**Comment by balintzs** [comment]
Ah, sorry I forgot about this one. Yes, go ahead. LGTM

**Pull request merged**